market/proposals/contact.go

```diff
@@ -15,7 +15,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package dto
+package proposals
```

Comment:
Golang code style recommends not use plural for package names. 
Let's call it just `proposal`.

Reply:
Good suggestion. But then here comes my question in PR description: which is better proposal.ServiceProposal or market.ServiceProposal ? I would go for the second

Reply:
Sorry, forget to answer the question. For me, `proposal.ServiceProposal` sounds better.

Reply:
I guess we will vote during standup :)